scripts/validate-tmpfs.sh

```
@@ -0,0 +1,62 @@
+  fi
+
+  echo "[${service}]"
+  for mount in ${SERVICES[$service]}; do
+    # df output: Filesystem Size Used Available Use% Mounted
+    df_line=$(docker exec "$container" df -h "$mount" 2>/dev/null | tail -1) || {
+      echo "  ${mount}: unable to read (no df in image?)"
+      continue
+    }
+    size=$(echo "$df_line" | awk '{print $2}')
+    used=$(echo "$df_line" | awk '{print $3}')
+    avail=$(echo "$df_line" | awk '{print $4}')
```

Shellcheck (0.11.0)

[warning] avail appears unused. Verify use (or export if used externally).

(SC2034)